build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
```

Comment:
are those enabled by default or could we simply drop this?

Reply:
They are required and cannot be dropped